.github/workflows/ci.yml

```diff
@@ -63,8 +63,11 @@ jobs:
         board: ['esp32', 'esp32s2', 'esp32s3', 'esp32c3']
         std: ['true', 'false']
     steps:
-      - name: Setup | cargo-generate
-        run: cargo install cargo-generate
+      - name: Install cargo-generate
```

Comment:
Why this and not just `cargo install cargo-generate`?

Reply:
I changed to the binary approach as it's way faster. If you think is better to do it with `cargo install` I am happy to undo this change